src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
I thought we were doing only one step after another one, I followed getEntityAllTypes
https://github.com/telefonicaid/fiware-orion/blob/develop/src/lib/serviceRoutinesV2/getEntityAllTypes.cpp#L59
which has an options=value possibility in api doc. As said in your PR #1171
"Good enough for now, until options itself is implemented.
options=values is not implemented."